GoogleForCreators/web-stories-wp
# Right Click Menus: remove unneeded actions

## Context

Had a meeting today with Andrew to discuss UX for the right click menu. Some actions were deemed confusing or unnecessary. This PR removes them.

[Doc](https://docs.google.com/document/d/1gcVTVMdGaM9Ds8T_EJAGQbiBlSArnJh_onuTBXmSISs/edit#heading=h.f930w6zw8ak)

## Summary

1. **Removing:**
- `Copy`
- `Paste`
- `Delete`
- `Replace Background Image`

And all tests associated with the above actions.

2. **Removes the page manipulation actions from the background media menu but not the page menu.** 

The page menu has nothing else, so I thought it would be a very sad empty menu without them.

3. Removes `lockAspectRatio` from the styles that can be copied, pasted, and cleared.


## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

||Text|Media|Background Media|Shapes|Page|
|--|--|--|--|--|--|
|Before|![image](https://user-images.githubusercontent.com/22185279/129269206-e45837ef-028e-44d3-a4cf-b6833d5e51b3.png)|![image](https://user-images.githubusercontent.com/22185279/129269185-c8bcfccb-ed91-405d-b372-1cc9c050755b.png)|![image](https://user-images.githubusercontent.com/22185279/129269158-eab3b3c0-2a47-4d58-95e2-89c98bc914c1.png)|![image](https://user-images.githubusercontent.com/22185279/129269117-469a9f86-7a99-4fa3-a914-aa375d9af940.png)|![image](https://user-images.githubusercontent.com/22185279/129269093-1f3503d8-3097-4557-8caf-64f03a4ee6e9.png)|
|After|![image](https://user-images.githubusercontent.com/22185279/129268862-f13366b7-65b1-494e-a6ce-5f5a8978512e.png)|![image](https://user-images.githubusercontent.com/22185279/129268945-b73043bd-1882-42b7-86c6-aa9eadb097ae.png)|![image](https://user-images.githubusercontent.com/22185279/129268969-58f3cec0-cfb8-4b0c-a3b3-af13ae42d6f8.png)|![image](https://user-images.githubusercontent.com/22185279/129268991-905f4d0b-4325-4ba7-87cf-5a504455b165.png)|![image](https://user-images.githubusercontent.com/22185279/129269023-5b249a90-c819-4613-874d-44000ab30822.png)|


## Testing Instructions

1. Turn on checklist flag
2. Go to editor
3. Try the right click menus on all the element types. Everything should work the same but some actions will not be there anymore. Should match the screenshots above.

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #
